kms/api-client/snippets_test.py

```diff
@@ -13,226 +11,410 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 
+import hashlib
+import os
+import pytest
```

Comment:
`pytest` is a 3rd party library, so please place it in the 2nd block.

```python
import hashlib
import os
import time
import uuid

import pytest

import create... # local imports
```

Reply:
Done